### ISSUE TITLE
Removing the only member in an IAM Policy binding causes Protobuf error

### ISSUE THREAD

**Issue by quartzmo** [googleapis/google-cloud-ruby#10576]
## Example:

With only one entry in bindings:

```
    policy = topic.policy do |p|
      p.remove "roles/owner", "user:owner@example.com"
    end
```

```
Google::Protobuf::TypeError: nil message not allowed here.
    /Users/quartzmo/code/google/codez/google-cloud-ruby/google-cloud-pubsub/lib/google/cloud/pubsub/policy.rb:162:in `initialize'
    /Users/quartzmo/code/google/codez/google-cloud-ruby/google-cloud-pubsub/lib/google/cloud/pubsub/policy.rb:162:in `new'
    /Users/quartzmo/code/google/codez/google-cloud-ruby/google-cloud-pubsub/lib/google/cloud/pubsub/policy.rb:162:in `to_grpc'
    /Users/quartzmo/code/google/codez/google-cloud-ruby/google-cloud-pubsub/lib/google/cloud/pubsub/topic.rb:803:in `update_policy'
    /Users/quartzmo/code/google/codez/google-cloud-ruby/google-cloud-pubsub/lib/google/cloud/pubsub/topic.rb:769:in `policy'
```

**Comment by quartzmo** [comment]
```
quartzmo@Chriss-iMac google-cloud-pubsub % bundle update
Fetching gem metadata from https://rubygems.org/..........
Resolving dependencies...
Using rake 13.0.3
Using public_suffix 4.0.6
Using ansi 1.5.0
Using ast 2.4.2
Using autotest-suffix 1.1.0
Using builder 3.2.4
Using bundler 2.2.7
Using concurrent-ruby 1.1.8
Using docile 1.3.5
Using faraday-net_http 1.0.1
Using multipart-post 2.1.1
Using ruby2_keywords 0.0.4
Using google-protobuf 3.15.5 (universal-darwin) (was 3.15.3)
Using os 1.1.1
Using memoist 0.16.2
Using jaro_winkler 1.5.4
Using jwt 2.2.2
Using rainbow 3.0.0
Using multi_json 1.15.0
Using google-cloud-errors 1.0.1 from source at `../google-cloud-errors`
Using unicode-display_width 1.6.1
Using minitest 5.14.4
Using path_expander 1.1.0
Using redcarpet 3.5.1
Using simplecov-html 0.12.3
Using simplecov_json_formatter 0.1.2
Using yard 0.9.26
Using addressable 2.7.0
Using faraday 1.3.0
Using parser 3.0.0.0
Using googleapis-common-protos-types 1.0.6
Using ruby-progressbar 1.11.0
Using parallel 1.20.1
Using grpc 1.36.0 (universal-darwin)
Using signet 0.15.0 (was 0.14.1)
Using google-cloud-env 1.4.0 from source at `../google-cloud-env`
Using minitest-server 1.0.6
Using minitest-focus 1.2.1
Using minitest-rg 5.2.0
Using simplecov 0.21.2
Using yard-doctest 0.1.17
Using googleapis-common-protos 1.3.11
Using googleauth 0.16.0 (was 0.15.1)
Using minitest-autotest 1.1.1
Using minitest-reporters 1.3.8
Using rubocop 0.74.0
Using grpc-google-iam-v1 0.6.11
Using google-style 1.24.0
Using gapic-common 0.4.0
Using google-cloud-pubsub-v1 0.3.0 from source at `../google-cloud-pubsub-v1`
Using google-cloud-core 1.5.0 from source at `../google-cloud-core`
Using google-cloud-pubsub 2.3.2 from source at `.`
Bundle updated!

```

**Comment by quartzmo** [comment]
This was closed by #10579.
